webcompat/templates/topbar.html

```diff
@@ -32,12 +32,17 @@
               <span class="wc-Navbar-link-label">Log in</span>
             </a>
             {% else %}
-            <a class="wc-Navbar-link js-login-link" href="{{ url_for('logout') }}">
-              <span class="wc-Navbar-link-icon wc-Icon wc-Icon--arrow-circle-right" aria-hidden="true"></span>
-              <span class="wc-Navbar-link-label">Logout</span>
-            </a>
+            <div class="r-ResetButton wc-Navbar-link js-login-link wc-DropdownHeader">
+              <button class="r-ResetButton" aria-pressed="false">
+                <img class="wc-Navbar-avatar" src="{{ session.avatar_url }}s=50" alt="User Avatar">
+                <span class="wc-Navbar-link-icon wc-Icon wc-Icon--chevron-down" aria-hidden="true"></span>
+              </button>
+              <div class="wc-DropdownHeader-content">
+                <a class="wc-Navbar-linkDropdown" href="{{ url_for('logout') }}">My activities</a>
+                <a class="wc-Navbar-linkDropdown" href="{{ url_for('logout') }}">Logout</a>
+              </div>
+            </div>
             {% if session.user_id and session.avatar_url %}
-            <img class="wc-Navbar-avatar" src="{{ session.avatar_url }}s=40" alt="User Avatar">
             {% endif %}
```

Comment:
We can kill this empty `{% if session.user_id and session.avatar_url %}{%endif%}` block now, I think. :hocho: